v12x/src/main/java/com/foursoft/harness/vec/v12x/navigations/PartOccurrenceOrUsageNavs.java

```diff
@@ -173,6 +180,25 @@ public static Function<VecOccurrenceOrUsage, Optional<VecTopologyNode>> topology
                 .collect(StreamUtils.findOneOrNone());
     }
 
+    public static Function<VecOccurrenceOrUsage, VecPrimaryPartType> primaryPartType() {
+        return occurrenceOrUsage -> {
+            if (occurrenceOrUsage instanceof VecPartOccurrence) {
+                return primaryPartTypeOfOccurrence().apply((VecPartOccurrence) occurrenceOrUsage);
+            }
+            return ((VecPartUsage) occurrenceOrUsage).getPrimaryPartUsageType();
+        };
+    }
```

Comment:
Even the original method was renamed from `primaryPartType` to `primaryPartTypeOfOccurrence`, it's not a breaking change since this method will be used instead now.